sale_financial_risk_pos_compatibility/static/src/js/CreateOrderPopup.esm.js

```diff
@@ -0,0 +1,73 @@
+/** @odoo-module **/
+
+import CreateOrderPopup from "point_of_sale.CreateOrderPopup";
+import Registries from "point_of_sale.Registries";
+
+export const CreateOrderPopupRisk = (CreateOrderPopup) =>
+    class CreateOrderPopupRisk extends CreateOrderPopup {
+        async _actionCreateSaleOrder(order_state) {
+            if (order_state === "draft") {
+                // Default behavior
+                return await super._actionCreateSaleOrder(order_state);
+            }
+            this.extraContext = false;
+            const order = this.env.pos.get_order();
+            const risk_amount =
+                order.get_total_with_tax() + order.get_rounding_applied();
+            const [partner] = await this.rpc({
+                model: "res.partner",
+                method: "read",
+                args: [
+                    order.partner.id,
+                    [
+                        "risk_exception",
+                        "risk_sale_order_limit",
+                        "risk_sale_order",
+                        "risk_sale_order_include",
+                        "risk_total",
+                        "credit_limit",
```

Comment:
Why not load this data when you load the POS?

Reply:
> Why not load this data when you load the POS?

Because partner fields value change at each order